### PR TITLE
Added Threat Actor to trustar_report definition

### PR DESCRIPTION
**Pull request by pwrenn**
Simple change adding Threat Actor to the attributes within trustar_report

### CODE REVIEW

**Comment by adulau** [comment]
Thanks for the PR.

A small question, is there a specific reason to not use a MISP galaxy to link with threat-actor? That would be more consistent within a sharing community.

**Comment by pwrenn** [comment]
@adulau TruSTAR declares Threat Actor as an indicator type like the rest of the attributes in our definition.json file. Without being able to declare it here it will not be ingested properly into MISP. I think this can run in parallel with a Galaxy.

**Comment by packet-rat** [comment]
Operations against Threat_Actor Attribute are failing because TruSTAR Report Object has reverted to the original version ( as of at least 2.4.135)

[JSON File] (https://github.com/MISP/misp-objects/blob/main/objects/trustar_report/definition.json)

**Comment by adulau** [comment]
Can someone update the PR? Thanks a lot. Not really familiar with TruSTAR.

**Comment by packet-rat** [comment]
@adulau -  Thanks for the timely response.  I'm reaching out to pwrenn at TruSTAR to see if we can fix the PR.

Question:  You referenced using a MISP Galaxy Reference.  Can you summarize an example and/or point to a good reference?

**Comment by adulau** [comment]
It should be fixed in https://github.com/MISP/misp-objects/commit/2cb16e7be06305a9da54661bd4c87d78215b8e85 - Feel free to reopen if you have any issue.
